app/src/main/java/com/x8bit/bitwarden/data/vault/repository/VaultRepositoryImpl.kt

```diff
@@ -320,6 +321,12 @@ class VaultRepositoryImpl(
             .syncFolderUpsertFlow
             .onEach(::syncFolderIfNecessary)
             .launchIn(ioScope)
+
+        databaseSchemeManager
+            .lastDatabaseSchemeChangeInstantFlow
+            .filterNotNull()
+            .onEach { syncIfNecessary() }
```

Comment:
Do we want to just call sync explicitly?

I don't think it will make a huge difference